RPT_Application/src/main/java/com/harman/report_database/Report.java

```diff
@@ -0,0 +1,121 @@
+package com.harman.report_database;
+
+import java.text.DecimalFormat;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import com.harman.user_database.Employee;
+
+@Entity
+@Table(name = "report")
+public class Report {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private long id;
+	private LocalDate date;
+	private String employeeUsername;
+	private String wwb;
+
+	@OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true, mappedBy = "report")
+	private List<Task> tasks = new ArrayList<Task>();
+
+	@ManyToOne(cascade = CascadeType.MERGE)
+	private Employee employee;
+
+	public Report() {
+	}
+
+	public Report(List<Task> list) {
+		this.tasks = list;
+	}
+
+	public long getId() {
+		return id;
+	}
+
+	public void setId(long id) {
+		this.id = id;
+	}
+
+	public String getTasks() {
+		String taskRepresentation = "";
+		for (Task t : this.tasks) {
+			taskRepresentation += ("(" + Long.toString(t.getId()) + " : ");
+			taskRepresentation += (t.getWwbType().toString() + ") | ");
+		}
+		return taskRepresentation;
+	}
+
+	public void setTasks(List<Task> tasks) {
+		this.tasks = tasks;
+	}
+
+	public Employee getEmployee() {
+		return employee;
+	}
+
+	public void setEmployee(Employee employee) {
+		this.employee = employee;
+		this.setEmployeeUsername(this.employee.getUsername());
+	}
+
+	public void addTask(Task task) {
+		this.tasks.add(task);
+	}
+
+	public LocalDate getDate() {
+		return date;
+	}
+
+	public void setDate(LocalDate date) {
+		this.date = date;
+	}
+
+	public String getEmployeeUsername() {
+		return employeeUsername;
+	}
+
+	public void setEmployeeUsername(String employeeUsername) {
+		this.employeeUsername = employeeUsername;
+	}
+
+	public String getWwb() {
+		int f = 0, b = 0, r = 0;
```

Comment:
Please do not use abbreviations. In code below it is not easy to guess what is f,b,r,t.
We can use i,j in case of counters and x,y,z in case of coordinates.